docs/rebuild_admin.rst

```diff
@@ -0,0 +1,404 @@
+Rebuilding an Admin Workstation USB
+-----------------------------------
+
+In cases where an Admin Workstation stick has been lost or destroyed, and no
+backup exists, it is possible to rebuild one. In order to do so, you'll need
+ 
+ - physical access to the SecureDrop servers
+ - 2 USB sticks:
+
+   - Tails Master USB
+   - 1 replacement Admin Workstation USB (USB3 and 16GB or better recommended)
+
+The process requires experience with the Linux command line and Tails, and 
+can take up to 3 hours. If a backup of the SecureDrop application server is available,
+reinstalling the instance may be simpler. An outline of the steps involved is as 
+follows:
```

Comment:
I would say "an outline of the steps involved in rebuilding an *Admin Workstation*", just to resolve ambiguity with the previous sentence, or move the "reinstalling may be simpler" sentence somewhere else.

Reply:
fixed